MaterialLib/SolidModels/Lubby2.h

```diff
@@ -72,10 +85,34 @@ struct LocalLubby2Properties
           etaM0(mp.etaM0(t, x)[0]),
           mK(mp.mK(t, x)[0]),
           mvK(mp.mvK(t, x)[0]),
-          mvM(mp.mvM(t, x)[0])
+          mvM(mp.mvM(t, x)[0]),
+          Tref(mp.Tref(t, x)[0]),
+          mGT(mp.mGT(t, x)[0]),
+          mKT(mp.mKT(t, x)[0]),
+          Q(mp.mKT(t, x)[0])
     {
     }
 
+    void update(double const s_eff, double const T)
+    {
+        double const dT(T - Tref);
+        double const GM0_s_eff = GM0 * s_eff;
+        GK = GK0 * std::exp(mK * GM0_s_eff);
+        etaK = etaK0 * std::exp(mvK * GM0_s_eff);
+        if (std::isnan(T))
+        {
+            etaM = etaM0 * std::exp(mvM * GM0_s_eff);
```

Comment:
Try to avoid any code duplication.
Especially this branch and all of the above computations are already implemented in the `update(s)` overload. Don't repeat them.

Reply:
Here we have a problem, in the function ```integrateStress``` ```update(s)``` was changed to ```update(s, T)``` for TM process, while for ```SmallDeformation``` ```T``` is ```nan```, so add a check point here... Thinking how to due with that...